config-EXAMPLE.ini

```diff
@@ -6,6 +6,7 @@ server = its.domain.name.com
 server_username = <name>
 
 [database]
+db_type = remote ; or local
```

Comment:
Instead, you can check if ssh exists in the config. If it is, it is certain that an SSH tunnel needs to be used.